### PR TITLE
db-facts CLI can output airflow connection URI string

### PR DESCRIPTION
**Pull request by austinweisgrau**

### CODE REVIEW

File: db_facts/exports.py [hunk 1]
```diff
@@ -1,18 +1,104 @@
...
+    """Generically translate credentials into airflow credential string."""
+    print(AirflowConnection(**exports).get_uri())
+
+
+class AirflowConnection:
+    """Copied from the apache-airflow airflow.operators.connection.Connection class
+
+    Simplified to reproduce a minimal implementation of the get_uri() method.
+    """
+
```
Comment (bluelabsbutler) [inline db_facts/exports.py:34]: [flake8] W293 blank line contains whitespace

File: db_facts/exports.py [hunk 2]
```diff
@@ -1,18 +1,104 @@
...
+        self.login = user or kwargs.pop('username')
+        self.password = password
+        self.schema = database
+        self.port = port
+        self.extra = kwargs
+
+    def get_uri(self) -> str:
+        """Return connection in URI format"""
+        if "_" in self.conn_type:
+            logging.warning(
```
Comment (bluelabsbutler) [inline db_facts/exports.py:67]: [flake8] F821 undefined name 'logging'

File: db_facts/runner.py [hunk 3]
```diff
@@ -69,6 +76,15 @@ def run(self, argv: List[str]) -> int:
                                          '"abc-dev-dbadmin")'))
             sh_parser.set_defaults(func=self.dump_sh)
 
+            airflow_parser = subparsers.add_parser('airflow',
+                                                   help=('Report output in Airflow connection '
+                                                         'string format'))
+            airflow_parser.add_argument('dbname', nargs=1,
+                                        help=('Friendly name of database '
+                                         '(e.g., "redshift", "dmv", '
```
Comment (bluelabsbutler) [inline db_facts/runner.py:84]: [flake8] E128 continuation line under-indented for visual indent

**Review by ryantimjohn — APPROVED** [review]

**Review by jessezlotoff — APPROVED** [review]

**Comment by austinweisgrau** [comment]
We are not merging this because we are instead using a lambda function to maintain a "mirror" of secrets in AWS secrets manager to use as a backend for airflow.

https://github.com/bluelabsio/secrets_manager_airflow_mirror

**Pull request closed without merging**